### ISSUE TITLE
Test the scripts on "dummy input"

### ISSUE THREAD

**Issue by lurch** [resin-io-modules/drivelist#139]
While reading through #138 it occurred to me that while we check that the JS is able to correctly parse example script output (i.e. convert YAML to JSON), we don't currently test the scripts themselves.
I dunno how difficult it would be to set up, but I guess it's conceivable that different versions of an OS could output info in (slightly?) different formats, so it might be nice if we could mock up "sample input" to test the drive scripts.

Alternatively, given how low-level the scripts operate, it may actually be impossible to "unit test" them (especially the Windows VBScript stuff) :-/
(this is just a quick suggestion, I haven't thought about it too deeply...!)

**Comment by jviotti** [comment]
Yeah, I've been wishing to test the scripts since ever, but I never found a good way to do so. We can of course mock some commands or files in the OS (with things like chroot I guess), but its definitely more complex for Windows.

Maybe @jhermsmeier has some thoughts? Manual testing seems to be the best we can do for something like this.
